File: Iniciante/1013/1013.cs
using System; 
using System.Linq;
class URI {

    static void Main(string[] args) { 

        int[] n = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();

int e = ((n[0] + n[1] + Math.Abs(n[0] - n[1])))*1/2;
int c = ((e + n[2] + Math.Abs(e - n[2])))*1/2;
Console.WriteLine(Math.Max(e, c) + " eh o maior");

    }

}

File: Iniciante/1013/1013.java
import java.io.IOException;
import java.util.Arrays;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
       
               Scanner nabana = new Scanner(System.in);
        int[] d = Arrays.stream(nabana.nextLine().split(" ")).mapToInt(Integer::parseInt).toArray();

        int n = maior(d[0], d[1]);
        n = maior(n,d[2]);
        System.out.println(n + " eh o maior");






    }
    public static int maior (int n1, int n2){
        int l = (n1+n2 + Math.abs(n1-n2))/2;
        return l;


    }
       
    
    }